contrib/platform/src/com/sun/jna/platform/win32/Ntddk.java

```diff
@@ -0,0 +1,67 @@
+package com.sun.jna.platform.win32;
+
+/**
+ * The interface for the Ntddk.h constants.
+ *
+ * @author Luke Quinane
+ */
+public class Ntddk {
+    public static final int FILE_DEVICE_8042_PORT = 0x00000027;
+    public static final int FILE_DEVICE_ACPI = 0x00000032;
+    public static final int FILE_DEVICE_BATTERY = 0x00000029;
+    public static final int FILE_DEVICE_BEEP = 0x00000001;
+    public static final int FILE_DEVICE_BUS_EXTENDER = 0x0000002a;
```

Comment:
- This should be an _interface_ (as indicated also by the Javadoc)
- Please drop the _public static final_ as this is the default for interfaces